src/tracer.js

```diff
@@ -472,6 +484,7 @@ module.exports.postTrace = function postTrace(traceObject) {
  */
 module.exports.sendTrace = function sendTrace(runnerUpdateFunc) {
     const tracerObj = module.exports.getTrace();
+    setLabelsToEvent(tracerObj);
```

Comment:
you can move this to where we set the ecs metadata instead of doing it twice (here and `sendCurrentTrace`)